### ISSUE TITLE
Continue mirroring this repo from the monorepo

### ISSUE THREAD

**Issue by powderluv** [ROCm/ROCm-Device-Libs#100]
Would it be possible to continue mirroring this repo so it allows end users to build with their own LLVM https://github.com/openxla/iree/pull/14894

**Comment by kzhuravl** [comment]
Yes, I will enable the mirroring today.

Note that this repo has been deprecated. device-libs, comgr, hipcc are now a part of llvm-project: https://github.com/RadeonOpenCompute/llvm-project/tree/amd-stg-open/amd. There is some scripting I need to enable to unmerge those 3 projects and mirror it here. It will be mirrored for some time.

**Comment by cgmb** [comment]
@kzhuravl, it's really comforting to hear that the mirror will be maintained for a while.

I'm thrilled with the effort going into upstreaming the components that are tightly coupled to LLVM internals (rocm-device-libs and comgr), since that will eventually everything simpler. However, there's a number of third-party packaging workflows built around llvm/llvm-project + separate repos, and the transition to llvm/llvm-project + ROCm/llvm-project is going to be a bit annoying to handle (especially because it's temporary and those workflows will need to change _again_ once the upstreaming is complete).

If you're able to maintain the mirror until the component has been upstreamed into llvm/llvm-project, it would be very, very much appreciated. It won't just save me a few hours of implementing the updated workflow, but it will save me hours of explaining it to other developers.

**Comment by kzhuravl** [comment]
We would like to archive device-libs and comgr projects.

In order to build device-libs and comgr, which are now integrated into https://github.com/ROCm/llvm-project/tree/amd-staging/amd, can you try the following steps:

- cd <your-choice-of-llvm>
- git remote add rocm-llvm https://github.com/ROCm/llvm-project.git
- git fetch rocm-llvm <branch-you-would-like-to-checkout> // you can use amd-staging as the branch, if building against trunk
- git checkout rocm-llvm/<branch-you-would-like-to-checkout> -- amd

You will now have "amd" directory in your fork of llvm.

Can you please give it a try?

**Comment by lamb-j** [comment]
We are now in the process of archiving this repository, and don't advise using this repository in builds moving forward.

If you have any problems with the above instructions to fetch the device libs from it's new location, can you open a new issue at https://github.com/ROCm/llvm-project with the 'device-libs' tag? Thanks!
